packages/aws-rfdk/lib/deadline/scripts/bash/installDeadlineRepository.sh

```diff
@@ -69,7 +71,17 @@ set +x
 
 INSTALLER_DB_ARGS_STRING=''
 for key in "${!INSTALLER_DB_ARGS[@]}"; do INSTALLER_DB_ARGS_STRING=$INSTALLER_DB_ARGS_STRING"${key} ${INSTALLER_DB_ARGS[$key]} "; done
-$REPO_INSTALLER --mode unattended --setpermissions false --prefix "$PREFIX" --installmongodb false --backuprepo false ${INSTALLER_DB_ARGS_STRING}
+
+REPOSITORY_SETTINGS_ARG_STRING=''
+if [ ! -z "$DEADLINE_REPOSITORY_SETTINGS_FILE" ]; then
+  if [ ! -f "$DEADLINE_REPOSITORY_SETTINGS_FILE" ]; then
+    echo "WARNING: Repository settings file was specified but is not a file: $DEADLINE_REPOSITORY_SETTINGS_FILE. Repository settings will not be imported."
```

Comment:
I wonder if we should make this case a fail. The warning's in the logs, but it looks like a successful deploy from the CDK CLI (and CloudFormation), but the customer would only know it's a fail if they look at the repository's cloud-init logs.

Reply:
That's a good point, changed this to fail the script